src/core/window.c

```diff
@@ -649,6 +650,24 @@ meta_window_class_init (MetaWindowClass *klass)
                   0,
                   NULL, NULL, NULL,
                   G_TYPE_NONE, 0);
+
+  /**
+   * MetaWindow::geometry-allocate:
+   * @window: A #MetaWindow
+   *
+   * This is emitted when a window is about to be resized,
+   * allowing connectors to override the size of that window if
+   * required. Use meta_window_expand_minimum_size to do that.
```

Comment:
This comment is outdated and mentions a method name that does not exist anymore.